projects/boringssl/build.sh

```diff
@@ -48,3 +46,20 @@ for F in $fuzzerFiles; do
     zip -j $OUT/${fuzzerName}_seed_corpus.zip $SRC/boringssl/fuzz/${fuzzerName}_corpus/*
   fi
 done
+
+rm -rf genfiles && mkdir genfiles && $SRC/LPM/external.protobuf/bin/protoc asn1_pdu.proto --cpp_out=genfiles --proto_path=$SRC/fuzzing/proto/asn1-pdu-proto/
+
+fuzzerLPMFiles=$(find $SRC/ -maxdepth 1 -name "*.cc")
+
+for F in $fuzzerLPMFiles
+do
+  fuzzerName=$(basename $F .cc)
```

Comment:
instead of naming the fuzz targets like `fuzz_certs` or `fuzz_pkcs12` let's name them `certs_lpm`, `pkcs12_lpm`, etc